frontend/src/components/patient/CreatePatientForm.js

```diff
@@ -366,84 +371,97 @@ function CreatePatientForm(props) {
                   />
                 )}
               </Field>
             </div>
             <Accordion>
-              <AccordionItem title="Emergency Contact Information">
-                <div className="inlineDiv">
-                  <Field name="patientContact.person.lastName">
-                    {({ field }) => (
-                      <TextInput
-                        value={
-                          values.patientContact?.person.lastName === undefined
-                            ? ""
-                            : values.patientContact?.person.lastName
-                        }
-                        name={field.name}
-                        labelText="Contact Last Name"
-                        id={field.name}
-                        className="inputText"
-                      />
-                    )}
-                  </Field>
-                  <Field name="patientContact.person.firstName">
-                    {({ field }) => (
-                      <TextInput
-                        value={
-                          values.patientContact?.person.firstName === undefined
-                            ? ""
-                            : values.patientContact?.person.firstName
-                        }
-                        name={field.name}
-                        labelText="Contact First Name"
-                        id={field.name}
-                        className="inputText"
-                      />
-                    )}
-                  </Field>
-                </div>
-                <div className="inlineDiv">
-                  <Field name="patientContact.person.email">
-                    {({ field }) => (
-                      <TextInput
-                        value={
-                          values.patientContact?.person.email === undefined
-                            ? ""
-                            : values.patientContact?.person.email
-                        }
-                        name={field.name}
-                        labelText="Contact Email"
-                        id={field.name}
-                        className="inputText"
-                      />
-                    )}
-                  </Field>
-                  <Field name="patientContact.person.primaryPhone">
-                    {({ field }) => (
-                      <TextInput
-                        value={
-                          values.patientContact?.person.primaryPhone ===
-                          undefined
-                            ? ""
-                            : values.patientContact?.person.primaryPhone
-                        }
-                        name={field.name}
-                        labelText={`Contact Phone: ${configurationProperties.PHONE_FORMAT}`}
-                        id={field.name}
-                        className="inputText"
-                      />
+              <AccordionItem title={intl.formatMessage({ id: "emergencyContactInfo.title" })}>
```

Comment:
Thanks @jona42-ui 
Can you still get rid of the formatting changes here ?? 
its even wrong indentation

Reply:
okay on it!